src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -535,6 +535,60 @@ An GoogleComputeEngine NodeSource can now be added using the *Create Node Source
 
 WARNING: When ProActive server is stopped, GoogleComputeEngine instances will be automatically deleted. And when ProActive server is restarted, the infrastructure will be recovered as per the previous settings. The required GoogleComputeEngine instances will be re-created and re-configured.
 
+==== Azure Infrastructure
+
+The Resource Manager allows to deploy ProActive nodes on the Microsoft Azure cloud. This infrastructure will create and manage virtual machines (VMs) with your custom Azure image and host ProActive nodes on them. Currently, the Azure infrastructure supports the creation of Linux VMs; support for Windows VMs is under development. In this section we will guide you through the requirements and configuration needed to setup an Azure infrastructure.
+
+===== Pre-Requisites
+
+To create an Azure infrastructure, ProActive relies on an existing Azure user account with a valid subscription. In particular, you will require:
+
+- *Azure account:* You should have a valid Azure account, linked to an Azure subscription with enough budget and core quota to deploy the VMs (and their underlying resources) that you request.
+- *Azure privileges:* If your account does not have full Admin privileges on your subscription, please make sure you have rights to:
+* Access a https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#resource-groups[resource group] of your choice.
+* Create VMs in that resource group.
+* Delete VMs in that resource group.
+- *Service Principal credentials:* ProActive uses Azure's API to manage the VMs in the infrastructure. This API requires a set of Active Directory _service principal credentials_ for non interactive login (client, secret, tenant). If you do not have them, you can create them via the https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal[Azure Portal] or https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest[Azure CLI]. Please ask your administrator in case you do not have the privileges to create them. The credentials should be strings of hexadecimal values like _1a2b3c4d-5e6f-1a2b-3c4d-5e6f-1a2b3c4d5e6f_.
+- *Custom Azure +image+:* ProActive will create the infrastructure's VMs from an Azure image. You will need to have a custom linux image available on the resource group of your choice. If you do not have one, you can create it by following https://docs.microsoft.com/en-us/azure/virtual-machines/linux/tutorial-custom-images[this tutorial].
+
+===== Infrastructure Configuration
+
+Once all the pre-requisites are met, you are ready to create an Azure infrastructure. On the ProActive's Resource Manager portal, select _AzureInfrastructure_ from the Infrastructure drop-down list.
+
```

Comment:
I think we might remove "On the ProActive's Resource Manager portal, select _AzureInfrastructure_ from the Infrastructure drop-down list." 
As this document could be used to create an Azure infrastructure through different ways: Resource Manager portal, command line or REST API.